omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -1375,6 +1388,15 @@
                 jQuery('#fifthSection').css('display','none');
                 jQuery('#sixthSection').css('display','block');
             }
+
+            var TBprophylaxisValue = jQuery("#TBprophylaxis option:selected").val();
+            if (TBprophylaxisValue == 'INH') {
+                jQuery('#INH').css('display','block');
```

Comment:
@reagan-meant Use class hidden and remove it to hide elements

Reply:
Fixed